src/benchmarklib/abstract_table_generator.cpp

```diff
@@ -15,25 +15,29 @@ AbstractTableGenerator::AbstractTableGenerator(const std::shared_ptr<BenchmarkCo
 void AbstractTableGenerator::generate_and_store() {
   Timer timer;
 
-  std::cout << "- Loading/Generating tables " << std::flush;
+  std::cout << "- Loading/Generating tables " << std::endl;
   auto table_info_by_name = generate();
-  std::cout << "(" << format_duration(std::chrono::duration_cast<std::chrono::nanoseconds>(timer.lap())) << ")"
-            << std::endl;
-
-  std::cout << "- Encoding tables " << std::flush;
 
   /**
    * Encode the Tables
    */
+  std::cout << "- Encoding tables if necessary" << std::endl;
```

Comment:
"if requested"?

Reply:
na, could be dictionary in the binary already. Then an encoding was requested, but there was no need to re-encode